### PR TITLE
Add a suggestion to try the beta channel

### PR DESCRIPTION
**Pull request by tidy-dev**
## Description
Add suggestion to try the beta channel in the about dialog. This suggestion will appear on any build other than beta.

### Screenshots

image

## Release notes
Notes: [Improved] Added link to beta download page in about dialog.

### CODE REVIEW

**Comment by tidy-dev** [comment]
Open to suggestion for copy.

Thought about trying to think of a quick forewarning like on the marketing page. "Feeling brave?", but it felt out of place here since they are not here to download desktop. This is linking to the beta landing page so maybe it isn't that necessary. Other thoughts were having the beta pill or beaker somewhere here.. but what I tried felt out of place.  

Other wording thoughts:
Same as marketing page: "Feeling brave?" -> next sentence "Try out the latest feature on the beta channel."
"Interested in the latest features?"
"Want to help test the latest features?"

**Comment by sergiou87** [comment]
I personally like the wording you chose. My only nit is maybe we should remove the `.` right after `Beta Channel`, since it looks a bit off to me. But even that you can ignore it 😂

File: app/src/ui/about/about.tsx [hunk 1]
```diff
@@ -227,6 +227,25 @@ export class About extends React.Component<IAboutProps, IAboutState> {
...
+
+    return (
+      <div>
+        <p className="no-padding">Looking for the latest features?</p>
+        <p className="no-padding">
+          Check out the{' '}
+          <LinkButton uri="https://desktop.github.com/beta">
+            Beta Channel
+          </LinkButton>
+          .
```
Comment (tidy-dev) [inline app/src/ui/about/about.tsx]:
```suggestion
```

**Comment by tidy-dev** [comment]
No period. :) 
image

**Review by KendallDoesCoding — COMMENTED** [review]
Great idea!

**Review by sergiou87 — APPROVED** [review]

**Pull request merged**